### PR TITLE
Porting backend to the WebAPI

### PR DESCRIPTION
**Pull request by joklek**
The backend and frontend was split. The server also takes in not strings but double as data, so lat, lon and radius should be sent as doubles. This frees us from some edge cases and unit tests.
What is new:
* InputDataValidator, which checks if lat, lon and radius is within accepted range.
* WebApi:GetBarsAround a new function which does the bar fetching.

### CODE REVIEW

File: BeerOverflowWindowsApp/MainWindow.cs [hunk 1]
```diff
@@ -140,13 +101,9 @@ private void ClearSelection()
...
                 }
             }
-            catch (ArgumentsForProvidersException)
-            {
-                MessageBox.Show("Please enter the required data correctly. Erroneus data is painted red.");
-            }
             finally
             {
-                HideProgressBars();
+                //HideProgressBars();
```
Comment (augisjay) [inline BeerOverflowWindowsApp/MainWindow.cs]: There shouldn't be commented lines of code
